source/extensions/load_balancing_policies/client_side_weighted_round_robin/client_side_weighted_round_robin_lb.h

```diff
@@ -99,18 +108,16 @@ class ClientSideWeightedRoundRobinLoadBalancer : public Upstream::ThreadAwareLoa
   // It stores the config necessary to calculate host weight based on the report.
   // The load balancer context stores a weak pointer to this handler,
   // so it is NOT invoked if the load balancer is deleted.
-  class OrcaLoadReportHandler : public LoadBalancerContext::OrcaLoadReportCallbacks {
+  class OrcaLoadReportHandler {
   public:
     OrcaLoadReportHandler(const ClientSideWeightedRoundRobinLbConfig& lb_config,
                           TimeSource& time_source);
-    ~OrcaLoadReportHandler() override = default;
-
-  private:
-    friend class ClientSideWeightedRoundRobinLoadBalancerFriend;
 
-    // {LoadBalancerContext::OrcaLoadReportCallbacks} implementation.
-    absl::Status onOrcaLoadReport(const OrcaLoadReportProto& orca_load_report,
-                                  const HostDescription& host_description) override;
+    // Update client side data from `orca_load_report`. Invoked from `onOrcaLoadReport` callback on
```

Comment:
Please update comment that it is invoked from `ClientSideHostLbPolicyData::onOrcaLoadReport`.